test/Validate.Capability.cpp

```diff
@@ -514,43 +516,26 @@ make_pair("OpDecorate %intt BuiltIn PatchVertices\n",             TessellationDe
 make_pair("OpDecorate %intt BuiltIn FragCoord\n",                 ShaderDependencies()),
 make_pair("OpDecorate %intt BuiltIn PointCoord\n",                ShaderDependencies()),
 make_pair("OpDecorate %intt BuiltIn FrontFacing\n",               ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn SampleId\n",                  ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn SamplePosition\n",            ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn SampleMask\n",                ShaderDependencies()),
+make_pair("OpDecorate %intt BuiltIn SampleId\n",                  vector<string>{"SampleRateShading"}),
+make_pair("OpDecorate %intt BuiltIn SamplePosition\n",            vector<string>{"SampleRateShading"}),
+make_pair("OpDecorate %intt BuiltIn SampleMask\n",                vector<string>{"SampleRateShading"}),
 make_pair("OpDecorate %intt BuiltIn FragDepth\n",                 ShaderDependencies()),
 make_pair("OpDecorate %intt BuiltIn HelperInvocation\n",          ShaderDependencies()),
 make_pair("OpDecorate %intt BuiltIn VertexIndex\n",               ShaderDependencies()),
 make_pair("OpDecorate %intt BuiltIn InstanceIndex\n",             ShaderDependencies()),
-// Though the remaining builtins don't require Shader, the BuiltIn keyword
-// itself currently does require it.
-make_pair("OpDecorate %intt BuiltIn NumWorkgroups\n",             ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn WorkgroupSize\n",             ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn WorkgroupId\n",               ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn LocalInvocationId\n",         ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn GlobalInvocationId\n",        ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn LocalInvocationIndex",        ShaderDependencies()),
```

Comment:
The above six cases disappeared, but I would have expected them to stay with `AllDependencies()` -- what am I missing?